1.0/good-objects/minimal_uppercase_digests/inventory.json

```diff
@@ -1,7 +1,7 @@
 {
   "digestAlgorithm": "sha512",
   "head": "v1",
-  "id": "ark:123/abc",
+  "id": "ark:00000/minimal_uppercase_digests",
```

Comment:
I am not seeing these `id` elements raised as warnings prior to this pull-request. Is there a reason that you decided to change the `id`s? ...just to make it a more accurate identifier for the name of the object?

Reply:
What are you using to validate that the value is a URI?
Given RFC3986, I would have thought that the `:` needs to be followed by a `/`:
https://tools.ietf.org/html/rfc3986#section-3

Reply:
Strictly speaking this is not relevant but it seems much nicer to have ids that give an indication of which object we are looking at, rather than `ark:123/abc` again and again

Reply:
Re validation of URIs -- see https://tools.ietf.org/html/rfc3986#appendix-A for gruesome ABNF. The general syntax requirement is letters + : + pretty much anything, certainly there are URI forms with no / after : including ARK (see e.g. https://tools.ietf.org/id/draft-kunze-ark-21.html#rfc.section.2.5.1)